sale_credit_point/models/sale_order.py

```diff
@@ -30,7 +31,9 @@ def credit_point_decrease_msg(self):
     def action_confirm(self):
         """Check credit before confirmation, update credit if check passed."""
         for sale in self:
-            sale.credit_point_check()
-            sale.partner_id.credit_point_decrease(
-                sale.amount_total, comment=self.credit_point_decrease_msg)
+            install_module = tools.config.get('init')
```

Comment:
this breaks std behavior and does not fix the issue, because here you can have 
```python
'init': {'sale_credit_point': 1, 'sale_stock': 1},
```
and tests will fail anyway.

Reply:
does not work and seems an hacked approach. I'm on it, stay tuned :)